src/provider/doc/ibmca-provider.man

```diff
@@ -30,8 +31,97 @@ specify:
+of
+.I mechanisms
+: \fBRSA\fP | \fBEC\fP | \fBDH\fP.
+If this option is not specified, \fBALL\fP is the default.
+.RE
+.PP
+.IP "debug = yes | no | stderr"
+.RS
+Enables debug output for the IBMCA provider. If this option is not specified,
+no debuging output is produced. If \fBdebug = stderr\fP is specified,
+debugging messages are printed to stderr. Otherwise the debug output is written
+into a trace file in \fB[/usr/local]/var/log/ibmca/trace-<provider-name>.<pid>\fP,
+where <provider-name> is the name of the IBMCA provider from the identity
+option, and <pid> is the process ID of the current process. You can also
+enable debugging by setting the environment variable \fBIBMCA_DEBUG\fP to
+\fBon\fP or \fBstderr\fP.
+.RE
+.PP
+.IP "fips = yes | no"
+.RS
+Enables FIPS mode for the IBMCA provider. If FIPS is enabled, it is ensured that
+the \fBlibica-cex\fP library is built with FIPS enabled, and the system is
+running in FIPS mode. If thats not the case, the IBMCA provider initialization
+fails, and a syslog message is issued. If the fips option is not specified, it
+is auto-detected if the \fBlibica-cex\fP library is built with FIPS enabled,
+and the system is running in FIPS mode. If both is true, then the FIPS mode is
+enabled for the IBMCA provider. If FIPS is enabled (either explicitly
+configured or due to auto-detection), the IBMCA provider registers its
+algorithm with \fBfips=yes\fP, thus the IBMCA algorithms may be fetched using
+a property query string containing \fBfips=yes\fP.
+.RE
+.PP
+fallback-properties = 
+.I property-query-string
+.RS
+A property query string that is used to fetch algorithms for fallback purposes.
+Fallbacks take place when the \fBlibica-cex\fP library does not support a
+certain key size or EC curve, or if the \fBlibica-cex\fP library fails to
+perform a crypto operation for whatever reason. If this option is not specified,
+a fallback property query string is built automatically as follows:
+\fBprovider!=<ibmca-provider-name>[,fips=yes]\fP. Clause \fBfips=yes\fP is
+included if FIPS mode is enabled for the IBMCA provider (either explicitly
+configured or due to auto-detection).
+Note that the IBMCA provider itself can not be used as fallback provider.
+Thus, the fallback property query string must either select a certain provider
+explicitly (e.g. \fBprovider=default\fP), or must exclude the IBMCA provider
+(\fBprovider!=<ibmca-provider-name>\fP). If FIPS mode is enabled for the
+IBMCA provider, the fallback provider should also be FIPS-enabled. Include
+\fBfips=yes\fP into the fallback property query string, or explicitly select
+the FIPS provider (\fBprovider=fips\fP).
+.RE
+.PP
+.SH ENVIRONMENT
+.TP
+.BR IBMCA_DEBUG
+If
+.B $IBMCA_DEBUG
+is set to \fBstderr\fP debug output to stderr for the IBMCA provider is enabled.
+If it is set to \fBon\fP the debug output is written into a trace file in
+\fB[/usr/local]/var/log/ibmca/trace.<pid>\fP, where <pid> is the process ID of
```

Comment:
This should be [`/usr/local]/var/log/ibmca/trace-<provider-name>.<pid>` like above.
Will fix this but wait with force-push for more updates.